client/state/comments/reducer.js

```diff
@@ -126,7 +127,24 @@ export function totalCommentsCount( state = {}, action ) {
 	return state;
 }
 
+/**
+ * Keeps track of if a comment list request is in flight
+ * @param {object} state  - global redux state
+ * @param {object} action - redux action
+ * @returns {boolean} new redux state
+ */
+export function isCommentListLoading( state = false, action ) {
+	switch ( action.type ) {
+		case COMMENTS_LIST_REQUEST:
+			return true;
+		case COMMENTS_RECEIVE: //we may need a more specific success/error action here
+			return false;
+	}
```

Comment:
Ideally, I'd like to show a loader with these conditions:
1. site is not available (handled as a ui check)
2. no comments are currently available for the view (eg it's ok to render stale data) AND we're making some effort to get this

I'm pretty open to ideas for figuring out how we keep that that we're making some sort of effort, and I'd be happy to try a few things, since I'm not sure we have a pattern for this yet for actions that use the data-layer.

@dmsnell So with the counter we'll want to also know when the request has failed, but I know you quite dislike the extra `_SUCCESS` and `_ERROR` actions. Without them though we can get into a state where one of the requests fails, and the others succeed so we never get back down to 0. Or similarly actions may fire for `COMMENTS_RECEIVE` in response to a post comments request, leaving us with a negative value.

We can certainly track this per site too, I was just being lazy at EOD to get something working.